### PR TITLE
feat: add indexes for attributes on metric tables

### LINKED ISSUE #18221: ClickHouse - No index for Metric Attributes
### Component(s)

ClickHouse exporter 0.70.0

### Is your feature request related to a problem? Please describe.

Hi, while we create an index for log attributes, we don't do it for metrics.

### Describe the solution you'd like

Create indexes for attributes in the same way as we do for traces and logs.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by StarpTech**
**Description:** <Describe what has changed.>

Fix #18221

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: exporter/clickhouseexporter/internal/gauge_metrics.go [hunk 1]
```diff
@@ -50,7 +50,13 @@ CREATE TABLE IF NOT EXISTS %s_gauge (
 		Value Float64,
 		SpanId String,
 		TraceId String
-    ) CODEC(ZSTD(1))
+    ) CODEC(ZSTD(1)),
+	INDEX idx_res_attr_key mapKeys(ResourceAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+	INDEX idx_res_attr_value mapValues(ResourceAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+	INDEX idx_scope_attr_key mapKeys(ScopeAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+	INDEX idx_scope_attr_value mapValues(ScopeAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+	INDEX idx_attr_attr_key mapKeys(Attributes) TYPE bloom_filter(0.01) GRANULARITY 1,
```
Comment (dmitryax) [inline exporter/clickhouseexporter/internal/gauge_metrics.go:58]: Should be `idx_attr_key` or `idx_dp_attr_key`?

Comment (StarpTech) [reply exporter/clickhouseexporter/internal/gauge_metrics.go]: Done

**Review by Frapschen — APPROVED** [review]
LGTM

**Review by atoulme — APPROVED** [review]
Approved.

**Comment by StarpTech** [comment]
@dmitryax ping.

**Review by dmitryax — APPROVED** [review]

**Pull request merged**